AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1377,6 +1408,19 @@ - (void)recursivelyClearFetchedData
 - (void)layout
 {
   ASDisplayNodeAssertMainThread();
+  
+  if (!_flags.isMeasured) {
```

Comment:
What should this method do if a superclass calls [super layout]?  Should this logic be run for a class that implements manual layout?